src/components/DocumentationTopic/OnThisPageRegistrator.vue

```diff
@@ -0,0 +1,96 @@
+<!--
+  This source file is part of the Swift.org open source project
+
+  Copyright (c) 2021-2022 Apple Inc. and the Swift project authors
+  Licensed under Apache License v2.0 with Runtime Library Exception
+
+  See https://swift.org/LICENSE.txt for license information
+  See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+-->
+
+<template>
+  <div />
```

Comment:
I'm not a big fan of an empty div... I feel the `OnThisPageRegistrator` could look more like a mixin than a component because it doesn't have any template reason to render or exist.

Thoughts?

Reply:
Mixins are the devil 😈  🤣

Reply:
Refactored to use a mixin instead. Let me know if its OK now.